src/parsers/manifest/dash/resolve_base_urls.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright 2015 CANAL+ Group
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import arrayIncludes from "../../../utils/array_includes";
+import resolveURL from "../../../utils/resolve_url";
+import { IBaseURL } from "./node_parsers/BaseURL";
+
+/**
+ * @param {Array.<string>} currentBaseURLs
+ * @param {Array.<Object>} newBaseURLs
+ * @returns {Array.<string>}
+ */
+export default function resolveBaseURLs(
+  currentBaseURLs : string[],
+  newBaseURLs : IBaseURL[]
```

Comment:
Do you mean here parent / children baseURLs ?

Reply:
No.
It's more
1. Already resolved baseURLs
2. New baseURLs encountered